perf/stability/alertmanager/prometheusrule.yaml

```diff
@@ -17,16 +17,21 @@ spec:
           expr: >
             sum(irate(istio_requests_total{reporter="destination", response_code=~"5.*"}[5m])) / sum(irate(istio_requests_total{reporter="destination"}[5m])) > 0.05
           for: 5m
+        - alert: LatencyHigh
+          expr: sum(irate(istio_request_duration_milliseconds_bucket{source_workload=~"svc.*"}[5m])) by (source_workload,namespace)  > 30000
+          for: 10m
+          annotations:
+            message:  "Request duration has slowed down for workload: {{`{{$labels.source_workload}}`}} in namespace: {{`{{$labels.namespace}}`}}. Response duration is {{`{{$value}}`}} seconds"
     - name: ./infra.rules
       rules:
         - alert: ContainerCPUUsageHigh
-          expr: (sum(rate(container_cpu_usage_seconds_total{namespace!="kube-system", container!="POD", namespace!=""}[5m])) BY (namespace, pod, container) * 100) > 80
+          expr: (sum(rate(container_cpu_usage_seconds_total{namespace!="kube-system", container=~"istio-proxy|discovery", namespace!=""}[5m])) BY (namespace, pod, container) * 100) > 80
           for: 5m
           annotations:
             summary: "Container CPU usage (namespace {{ $labels.namespace }}) (pod {{ $labels.pod }}) (container {{ $labels.container }})"
             description: "Container CPU usage is above 80%\n  VALUE = {{ $value }}\n"
         - alert: ContainerMemoryUsageHigh
-          expr: (sum(container_memory_usage_bytes{namespace!="kube-system", container!="POD", namespace!=""}) BY (container, pod, namespace)  / (sum(container_spec_memory_limit_bytes{namespace!="kube-system", container!="POD"}) BY (container, pod, namespace) > 0)* 100) > 80
+          expr: (sum(container_memory_usage_bytes{namespace!="kube-system", container=~"istio-proxy|discovery", namespace!=""}) BY (container, pod, namespace)  / (sum(container_spec_memory_limit_bytes{namespace!="kube-system", container!="POD"}) BY (container, pod, namespace) > 0)* 100) > 80
```

Comment:
use `container_memory_working_set_bytes` instead.

If you can also update it in our grafana dashaboards, that would be great.

Reply:
ok, I did some research and it seems like OOMKiller is tracking container_memory_working_set_bytes actually, updated